mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -38,6 +38,8 @@
 import static org.jooq.impl.DSL.selectDistinct;
 import static org.jooq.impl.DSL.table;
 import static org.jooq.impl.DSL.trueCondition;
+//import static org.folio.rest.jooq.routines.DeleteOldMarcIndexersVersions;
```

Comment:
Remove this.

Reply:
@okolawole-ebsco  Thanks! It is already merged.